addons/breathing/functions/fnc_fullHealLocal.sqf

```diff
@@ -145,3 +145,59 @@ _unit setDamage 0;
 [QACEGVAR(medical,FullHeal), _unit] call CBA_fnc_localEvent;
 _state = [_unit, ACEGVAR(medical,STATE_MACHINE)] call CBA_statemachine_fnc_getCurrentState;
 TRACE_1("after FullHeal",_state);
+
+/// Clear Stamina & weapon sway
+if (ACEGVAR(advanced_fatigue,enabled)) then {
+    
+    ["PDF"] call ACEFUNC(advanced_fatigue,removeDutyFactor);
+	["EDF"] call ACEFUNC(advanced_fatigue,removeDutyFactor);
+    ["LSDF"] call ACEFUNC(advanced_fatigue,removeDutyFactor);
+    ACEGVAR(advanced_fatigue,swayFactor) = EGVAR(pharma,originalSwayFactor);
+
+} else {
+
+    _unit enableStamina true;
+	_unit setAnimSpeedCoef 1;
+    _unit setCustomAimCoef 1;
+
+};
+
+/// Clear chroma effect & camera shake
+
+resetCamShake;
+["ChromAberration", 200, [ 0, 0, true ]] spawn
```

Comment:
why do you create a new PPEffect when you get healed?

Reply:
If you create a new PPEffect, with the same name as the other one that I am using (Ketamine, Fentanyl & Pervitin) it will as  well delete the one that has been spawned in via x medication. That is the only way I found to delete a already existing effect by basically spawning in a new one and deleting both at the end.

Reply:
For everyone reading this in future:

if you create a ppEffect with the same ID (in this case 200) it will override the old one

Reply:
maybe not delete but you can disable PPEffects like that (tho it will require more changes) https://github.com/acemod/ACE3/blob/952375c67639890bc5e92fdeea4967c89dc7fc85/addons/medical_feedback/functions/fnc_effectBloodVolume.sqf#L23

Reply:
yea, I know about disabling but I don't really think that this is the right way to do here. I'm not sure if it will drain resources if effect after effect stacks and just gets disabled instead of completely deleted. IDK deleting is just the best option we have in my opinion.